addons/website_sale/views/product_product_add.xml

```diff
@@ -1,33 +1,17 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <record id="product_product_view_form_add" model="ir.ui.view">
-        <field name="name">product.product.view.form.add</field>
+    <record id="product_product_view_form_normalized_website_sale" model="ir.ui.view">
+        <field name="name">product.product.view.form.normalized.website.sale.inherit</field>
         <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_product_view_form_normalized"/>
         <field name="arch" type="xml">
-            <form js_class="website_new_content_form">
-                <group name="pricing">
-                    <field name="website_url" invisible="1"/>
-                    <field name="company_id" invisible="1"/>
-                    <field name="currency_id" invisible="1"/>
-                    <field name="cost_currency_id" invisible="1"/>
-                    <field name="name" placeholder="e.g. Cheese Burger" string="Product Name"/>
-                    <field name="barcode"/>
-                    <field name="image_1920" invisible="1"/>   <!-- Hide the image field for set data in product barcodelookup -->
-                    <field name="description" invisible="1"/>  <!-- Hide the description field for set data in product barcodelookup -->
-                    <field name="weight" invisible="1"/>       <!-- Hide the weight field for set data in product barcodelookup -->
-                    <field name="categ_id" invisible="1"/>     <!-- Hide the categ_id field set data in product barcodelookup -->
-                    <label for="list_price" class="mt-1"/>
-                    <div name="Sales Price">
-                        <field name="list_price" class="oe_inline" widget="monetary"
-                               options="{'currency_field': 'currency_id', 'field_digits': True}"/>
-                        <field name="tax_string"/>
-                    </div>
-                    <field name="taxes_id" widget="many2many_tags"
-                           context="{'default_type_tax_use': 'sale', 'search_default_sale': 1}"
-                           options="{'create': false, 'create_edit': false}"/>
-                </group>
-            </form>
+            <xpath expr="//form" position="attributes">
+                <attribute name="js_class">website_new_content_form</attribute>
+            </xpath>
+            <xpath expr="//field[@name='company_id']" position="before">
+                <field name="website_url" invisible="1"/>
```

Comment:
it's better to add `fieldDependencies` on the widget.

Reply:
Yes, the xpath is not needed with `fieldDependencies`. I can help you with that if you'd like. 

You can already take a look at this PR: https://github.com/odoo/odoo/pull/167023/files#diff-c4a2967ba5565c44579ea72b23a00c03543543f5b207c996a6c636ece0deea1dR266-R270. We add `fieldDependencies` in `sol_product_many2one` and remove them from the xml.

Reply:
I tried and it doesn't work. Maybe fieldDependencies only for for the field and view_widget registries.